### ISSUE TITLE
0.52.4 version of crate-admin incorrectly checks discovery.zen.minimum_master_nodes

### ISSUE THREAD

**Issue by reversefold** [crate/crate-admin#177]
In crate 0.52.4 I'm seeing this error in the crate-admin:
```
The value of the cluster setting 'discovery.zen.minimum_master_nodes' needs to be greater than half of the maximum/expected number of nodes and equal or less than the maximum/expected number of nodes in the cluster.
```
But this is incorrect. This needs to be set to greater than half of the *master* nodes not the full number of nodes. As you can see from the output of `/_cat/nodes` I have 4 data nodes and 3 masters, so a value of 2 is correct.
```
11c1139047b3 172.17.0.3 1 31 0.84 d - Araña             
c95bbf6933b2 172.17.0.4 1 31 0.84 - m Nova-Prime        
eb00d0c09c4e 172.17.0.5 4 31 0.84 d - Gabriel Summers   
9f8829792a70 172.17.0.7 1 31 0.84 - * Naga              
73ddfc2ed47d 172.17.0.2 2 31 0.84 d - Achilles          
18a9f46e5512 172.17.0.6 4 31 0.84 - m Carnivore         
078c2ee20312 172.17.0.8 4 31 0.84 d - Beautiful Dreamer 
```

**Comment by joemoe** [comment]
thanks @reversefold, we will look into this.

**Comment by joemoe** [comment]
this is actually a bug, we will fix this today.

**Comment by kovrus** [comment]
@joemoe @reversefold 
 The bug fix is on master and backported to 0.52 and 0.54, but not yet released

**Comment by joemoe** [comment]
hi @reversefold, could you check if this is ok in 0.54.4, as it is released in 0.54.3.

**Comment by reversefold** [comment]
Looks good.
